nucleus/validate/scenario_test.py

```diff
@@ -224,6 +243,7 @@ def upload_external_evaluation_results(
             "overall_metric": aggregate_weighted_sum / aggregate_weight,
             "model_id": model_id,
             "slice_id": self.slice_id,
+            "level": "item" if is_item_eval else "scene",
```

Comment:
could we use the EntityLevel enum for this, also it could be cleaner to remove is_item_eval and change the var to the level of the result (which would get rid of this if statement here)

Reply:
Yep sg!